test/test_app_views.py

```diff
@@ -87,10 +87,17 @@ def test_retrieve_single_event(self):
         self.assertEqual(resp.status_code, 201)
         result = self.client().get(
             '/api/events/{}/'.format(1),
-            headers=dict(Authorization="Bearer " + access_token),)
+            headers=dict(Authorization="Bearer " + access_token))
         self.assertEqual(result.status_code, 200)
         self.assertIn('Barbecue', str(result.data))
 
+    def test_failed_retrieve_single_event(self):
+        """Test retrieving non existing event"""
+        resp = self.create_event()
+        self.assertEqual(resp.status_code, 201)
```

Comment:
Remove this line since it's irrelevant, the rest would be fine, and the test will be atomic